### PR TITLE
MM-60141 - Scheduled post copy text option

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Added option for copying raw text of a scheduled post irrespective of the error code.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-60141

#### Release Note
```release-note
None
```

### CODE REVIEW

**Review by yasserfaraazkhan — APPROVED** [review]
Thank you Harshil 🎉 
If you get a chance please add and e2e test case

**Review by pvev — APPROVED** [review]
LGTM! thanks. @harshilsharma63

**Comment by harshilsharma63** [comment]
> Thank you Harshil 🎉 If you get a chance please add and e2e test case

On it!

**Review by hmhealey — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]

File: webapp/channels/src/components/drafts/draft_row.tsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ import PlaceholderScheduledPostsTitle
 import EditPost from 'components/edit_post';
 
 import Constants, {StoragePrefixes} from 'utils/constants';
+import * as Utils from 'utils/utils';
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/drafts/draft_row.tsx:40]: why import everything, why not selectively import copyToClipboard

Comment (harshilsharma63) [reply webapp/channels/src/components/drafts/draft_row.tsx:40]: Done.

**Pull request merged**